src/widgets/ActivityDetails/ui/ActivityItemList.tsx

```diff
@@ -1,19 +1,37 @@
+import { ActivityEvents } from "../model/hooks"
+import { useActivityInProgress } from "../model/hooks/useActivityInProgress"
+
 import { ActivityDetails } from "~/entities/activity"
-import { ActivityCardItemList } from "~/entities/item"
+import { ActivityCardItemList, ItemCardButtonsConfig } from "~/entities/item"
 
 interface ActivityItemListProps {
   activityDetails: ActivityDetails
+  activityEvents: ActivityEvents[]
 }
 
-export const ActivityItemList = ({ activityDetails }: ActivityItemListProps) => {
+export const ActivityItemList = ({ activityDetails, activityEvents }: ActivityItemListProps) => {
+  const { activityInProgress, items } = useActivityInProgress(activityDetails, activityEvents)
+
   const isOnePageAssessment = activityDetails.showAllAtOnce
   const isSummaryScreen = false // Mock
+
+  const buttonsConfig: ItemCardButtonsConfig = {
+    isOnePageAssessment,
+    isBackShown: activityDetails.items.length > 1,
+    isSubmitShown: isOnePageAssessment && activityDetails.items.length === activityInProgress?.answers.length,
+    isSkippable: activityDetails.isSkippable,
+    isNextDisable: true, // Default value === TRUE  (Condition if answer value empty or not exist)
+  }
```

Comment:
This config should be applicable for every activity item. But but not for the whole activity unless it is a part of an activity flow

Reply:
This config is created on the top level and provided to children. In children  components, config can be changed regarding item settings.

Reply:
I am working on it in another branch. This will be fixed in the next MR